pkg/config/loader.go

```diff
@@ -55,7 +55,7 @@ func LoadConfig() (*Config, error) {
 	}
 	defer jsonFile.Close()
 
-	b, err := ioutil.ReadAll(jsonFile)
+	b, _ := ioutil.ReadAll(jsonFile)
```

Comment:
Error handling: the error should not be silenced here.

Reply:
Same thing here, let's add the error handling logic needed.  Probably should add something like this

```go
b, err := ioutil.ReadAll(jsonFile)
if err != nil {
   return nil, errors.Wrap(err, "read config")
}